nixos/modules/config/no-x-libs.nix

```
@@ -59,6 +59,9 @@ with lib;
       pinentry = super.pinentry.override { enabledFlavors = [ "curses" "tty" "emacs" ]; withLibsecret = false; };
       qemu = super.qemu.override { gtkSupport = false; spiceSupport = false; sdlSupport = false; };
       qrencode = super.qrencode.overrideAttrs (_: { doCheck = false; });
+      qt5 = super.qt5.overrideScope' (self: super: {
```

We should name nested self and super something different like self'/super' to not shadow the outer variables.

---

How can I fix this PR?

---

Update…
```suggestion
      qt5 = super.qt5.overrideScope' (self': super': {
```
…and the references to `self'`/`super'`

---

Error:
```
error: attribute 'qtbase' missing

       at /home/lafiel/works/src-nix/nixpkgs/nixos/modules/config/no-x-libs.nix:63:18:

           62|       qt5 = super.qt5.overrideScope' (self': super': {
           63|         qtbase = super.qtbase.override { withGtk3 = false; };
             |                  ^
           64|       });
       Did you mean one of hbase, qtpass or xbase?
(use '--show-trace' to show detailed location information)

```

---

> …and the references to self'/super'

Make it:
           63|         qtbase = super'.qtbase.override { withGtk3 = false; };
           
The `'` is parsed as just another letter in the `super'` identifier.

---

I'll make a PR for this